### PR TITLE
pmbootstrap: update to 1.30.0

### PR DESCRIPTION
**Pull request by tmahlburg**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/pmbootstrap/template [hunk 1]
```diff
@@ -10,7 +10,8 @@ maintainer="bra1nwave <bra1nwave@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://postmarketos.org"
 distfiles="https://gitlab.com/postmarketOS/pmbootstrap/-/archive/${version}/pmbootstrap-${version}.tar.gz"
-checksum=7288217212dbb89d6623718561440c3593a99c298db631912ea4a192ca397fb2
+checksum=b7737afd92ff2e527775556a3f824aa97b64dbf2012de827a8c3dd4b3b1731ba
+
```
Comment (ericonr) [inline srcpkgs/pmbootstrap/template:14]: This newline shouldn't be here.

**Comment by ericonr** [comment]
Also, why not 1.32.0?

File: srcpkgs/pmbootstrap/template (hunk 1 above)
Comment (tmahlburg) [reply srcpkgs/pmbootstrap/template]: Ah, yes, must have missed that. It's fixed now.

**Comment by tmahlburg** [comment]
As far as I can see here in their [GitLab](https://gitlab.com/postmarketOS/pmbootstrap/-/releases), 1.30.0 is the newest release.

**Comment by ericonr** [comment]
Weird, https://gitlab.com/postmarketOS/pmbootstrap/-/tags shows two more tags.

**Comment by tmahlburg** [comment]
Oh, okay. Well, now it's updated.

**Pull request closed without merging**